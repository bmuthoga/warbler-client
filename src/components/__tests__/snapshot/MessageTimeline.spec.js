import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import MessageTimeline from '../../MessageTimeline'

const props = {
  profileImageUrl: 'https://lorempixel.com/200/200/',
  username: 'testemail'
}

describe('MessageTimeline Snapshot Tests', () => {
  it('should render the MessageTimeline component', () => {
    const wrapper = shallow(
      <MessageTimeline { ...props } />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })  
})
