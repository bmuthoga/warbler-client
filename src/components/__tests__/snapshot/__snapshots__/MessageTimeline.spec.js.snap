// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageTimeline Snapshot Tests should render the MessageTimeline component 1`] = `
<div
  className="row"
>
  <UserAside
    profileImageUrl="https://lorempixel.com/200/200/"
    username="testemail"
  />
  <Connect(MessageList) />
</div>
`;
